### DISCUSSION TITLE
"Auto" language option for automatic language selection

### DISCUSSION THREAD

**Discussion by Andre601** [giscus/giscus#362] [Ideas]
It seems like that https://giscus.app automatically selects the language your browser/system uses, when available by the app.

Now I wonder why this isn't a thing yet in the main app itself? It could help people to understand what this text box under that one blog post is meant to be.
It would be a useful addition to have.

**Comment by laymonage** [comment]
> It seems like that https://giscus.app automatically selects the language your browser/system uses, when available by the app.

Oh, I hadn't even noticed that :astonished:

> Now I wonder why this isn't a thing yet in the main app itself? It could help people to understand what this text box under that one blog post is meant to be.
It would be a useful addition to have.

Hmm, let me see. If you remove `data-lang` from the `<script>` tag, does it auto detect the language? I haven't got the time to test it yet.

**Reply by Andre601** [reply]
I quickly tried this, but there is no change. The comment box appears in english.

**Reply by laymonage** [reply]
Thanks, I'll try to take a look into it this weekend.
